feature-libs/product/configurators/rulebased/cpq/cpq-configurator-rest.service.ts

```diff
@@ -39,6 +39,21 @@ export class CpqConfiguratorRestService {
     );
   }
 
+  readConfiguration(
+    configId: string,
+    _groupId: string // not yet implemented
```

Comment:
No underscore **_groupId** is used for parameters. Please remove it

Reply:
I need the underscore as, it is not yet implemented/used. Otherwise i get an error that the variable is not used.
or do you know other way, how this can be fixed/handled?